### ISSUE TITLE
stories

### ISSUE THREAD

**Issue by ben-domingue** [ben-domingue/irw#659]
https://osf.io/5qxkh/
https://link.springer.com/article/10.3758/s13428-024-02480-7#data-availability

**Comment by ben-domingue** [comment]
email permission 12-13-2024

**Comment by ben-domingue** [comment]
this is good data! 
are these imputed? i am thinking we should NA? 
```
[1] "idcr_martinez_2023_raco.Rdata"
       id item resp
1 cps2002   X5  1.0
2 cps2002   X6  1.0
3 cps2002   X7  1.0
4 cps2002   X8  1.0
5 cps2002   X9  1.0
6 cps2002  X10  0.5

                0               0.2              0.25 0.333333333333333               0.4 
              518               132               104               147                98 
              0.5               0.6 0.666666666666667              0.75               0.8 
              265                78               113                83                55 
                1 
             1203 
```

**Comment by KingArthur0205** [comment]
> this is good data! are these imputed? i am thinking we should NA?
> 
> ```
> [1] "idcr_martinez_2023_raco.Rdata"
>        id item resp
> 1 cps2002   X5  1.0
> 2 cps2002   X6  1.0
> 3 cps2002   X7  1.0
> 4 cps2002   X8  1.0
> 5 cps2002   X9  1.0
> 6 cps2002  X10  0.5
> 
>                 0               0.2              0.25 0.333333333333333               0.4 
>               518               132               104               147                98 
>               0.5               0.6 0.666666666666667              0.75               0.8 
>               265                78               113                83                55 
>                 1 
>              1203 
> ```

Yes this is imputed. I think I actually changed the dataset in my workspace but forgot to save it. Here is the updated csv file and thanks for the QC! This is why we need it :)

[idcr_martinez_2023_raco.csv](https://github.com/user-attachments/files/20352661/idcr_martinez_2023_raco.csv)

**Comment by ben-domingue** [comment]
i think this got corrupted @KingArthur0205 ?

```
bdomingu@gse7527517:~/Downloads$ unzip idcr_martinez_2023_raco.csv 
Archive:  idcr_martinez_2023_raco.csv
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of idcr_martinez_2023_raco.csv or
        idcr_martinez_2023_raco.csv.zip, and cannot find idcr_martinez_2023_raco.csv.ZIP, period.
```

**Comment by ben-domingue** [comment]
can you see i was only half paying attention ;)
also, i am somewhat opposed to spaces in table names. can i add a '_' in place of ' '

**Comment by ben-domingue** [comment]
also should the story recall ones be separate tables?

**Comment by KingArthur0205** [comment]
Certainly :)

**Comment by KingArthur0205** [comment]
> also should the story recall ones be separate tables?

I think they are already seperate?

**Comment by ben-domingue** [comment]
> > also should the story recall ones be separate tables?
> 
> I think they are already seperate?

apologies, i meant ' not separate' tables. are these separate constructs?

**Comment by ben-domingue** [comment]
@KingArthur0205 : should we collapse the three 'story recall' tables? 
@saviranadela : maybe someone from our summer team can add entries to the dictionary?
